torch/cuda/__init__.py

```diff
@@ -349,6 +349,12 @@ def synchronize():
     return torch._C._cuda_synchronize()
 
 
+def ipc_collect():
+    r"""Force collects GPU memory after it has been released by CUDA IPC."""
```

Comment:
I'd probably bulk the documentation up here a little more. When would I want to call this? What exactly does it do?